data/common.yaml

```diff
@@ -16,6 +16,9 @@ unbound::record: {}
 unbound::access:
   - '::1'
   - '127.0.0.1/8'
+unbound::do_not_query_address:
```

Comment:
What does it mean to have this default in the config?  Since you're chaning the default behavior, I'll need an appropriate version bump here, but more than that, I'd like to understand what it means to include both the local addresses to the `do_not_query_address`.